translations/en/module_en_lang.php

```diff
@@ -176,7 +176,7 @@
     'wd_transactionType' => 'Transaction Type',
     'wd_transaction_details_title' => 'Transaction Details',
     'wd_transaction_response_details' => 'Response Details',
-    'wd_trusted_shop_terms' => 'I agree that the data which are necessary for the liquidation of purchase on account and which are used to complete the identity and credit check are transmitted to payolution. My <u><a href="%s" target="_blank">consent</a></u> can be revoked at any time with effect for the future.',
+    'wd_trusted_shop_terms' => 'I agree that the data which are necessary for the liquidation of purchase on account and which are used to complete the identity and credit check are transmitted to Payolution. My <u><a href="%s" target="_blank">consent</a></u> can be revoked at any time with effect for the future.',
```

Comment:
This key needs to be changed to 'wd_payolution_terms' disscussed with @hbrandl

Reply:
ok, changed it